src/genie/libs/parser/ios/asr901/show_environment.py

```diff
@@ -0,0 +1,170 @@
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Or, Optional
+
+# ==============================
+# Schema for 'show environment'
+# ==============================
+class ShowEnvironmentSchema(MetaParser):
+
+    ''' Schema for "show environment" '''
+
+# These are the key-value pairs to add to the parsed dictionary
+    schema = {
+        'PowerSupply': {
```

Comment:
keyname is supposed to be lowercases. 
```suggestion
        'power_supply': {
```

Reply:
confirmed the change.